Finjector.Web/ClientApp/src/components/Entry/EntryEditButtons.tsx

```diff
@@ -14,110 +8,57 @@ import {
   faClone,
 } from "@fortawesome/free-solid-svg-icons";
 
-const landingPage = "/landing";
-
 interface Props {
   chartData: ChartData;
   savedChart: Coa;
+  save: () => void;
+  copy: () => void;
+  remove: () => void;
+  use: () => void;
```

Comment:
what does this get us? seemed cleaner before w/o passing along function callbacks, so now multiple levels need to be aware of how to handle the buttons, and if a new button is added/removed

Reply:
i thought it was clearer to have the mutations all in one place, it was easy to miss adding toasts when they were in different places. but heard, i think i'll just move them all into the one component.

Reply:
to me a "Button" component should be just a button, i wouldn't expect it to also be doing the work of mutations